modules/floating-buttons/assets/js/frontend/handlers/contact-buttons.js

```diff
@@ -342,8 +342,10 @@ export default class ContactButtonsHandler extends Base {
 			}
 		}
 
-		// Default to open in Editor
-		if ( elementorFrontend.isEditMode() ) {
+		if (
+			elementorFrontend.isEditMode() &&
+			'floating-buttons' === elementor?.config?.document?.type
```

Comment:
Why do we need to check the doc type here?

Reply:
Do you mean it renders in the Editor as well? When you are editing another page?

Reply:
the problem is that it renders as open, while normally on the FE it is not open. the only case when it hsould be open by default is when editing it as a floting button doc